### PR TITLE
Next Migration tool

### PR DESCRIPTION
**Pull request by lottamus**
- Closes https://github.com/stoplightio/platform-internal/issues/7168

### CODE REVIEW

File: 6.-migrations/next.md [hunk 1]
```diff
@@ -17,89 +17,68 @@ This shift leads to a number of benefits:
...
 
-#### Invite Your Team
+The NEXT Migration tool can migrate all of your NEXT projects, including all versions, OpenAPI files, Markdown files, and the most recently published Hub file.
 
-The fastest way to get your team moved over to Stoplight Platform is to [configure an email domain](../2.-workspaces/d.inviting-your-team.md#make-your-workspace-discoverable) and share a link to your workspace's signup page.
+However, there are a some items that won't be migrated:
 
-Alternatively, you can also [invite members](../2.-workspaces/d.inviting-your-team.md) individually using their email.
+- **Members** - the roles and permission system is slightly different in Stoplight Platform (see FAQ below), so Stoplight won't be able to migrate your teammates. You can [configure an allowed email domain](../2.-workspaces/allowed-email-domains.md) so your teammates can automatically join your workspace or [invite them manually by email](../2.-workspaces/d.inviting-your-team.md).
+- **Teams** - Stoplight is currently developing [Groups](https://stoplightio.productboard.com/portal/4-stoplight-platform/tabs/b52a696d-dde6-48ca-a1b4-deacf9610b4e/features/5739494/portal/expanded) functionality.
```
Comment (marbemac) [inline 6.-migrations/next.md:55]: `https://roadmap.stoplight.io/c/137-workspace-groups`

File: 6.-migrations/next.md [hunk 2]
```diff
@@ -17,89 +17,68 @@ This shift leads to a number of benefits:
...
 
-Now, let's create a Stoplight Workspace and add your projects.
+## Limitations
 
-1. [Create](https://stoplight.io/welcome/create) your new [Stoplight Workspace](../2.-workspaces/a.creating-a-workspace.md).
-2. Please follow this [step-by-step guide](../1.-quickstarts/add-projects-quickstart.md#connect-an-existing-git-project) to connect your VCS account and add your new projects.
+### Migration tool
 
-#### Invite Your Team
+The NEXT Migration tool can migrate all of your NEXT projects, including all versions, OpenAPI files, Markdown files, and the most recently published Hub file.
```
Comment (marbemac) [inline 6.-migrations/next.md:50]: Is it always doing most recently published now?

File: 6.-migrations/next.md [hunk 3]
```diff
@@ -17,89 +17,68 @@ This shift leads to a number of benefits:
...
-
-If neither of the above are relevant to your use case, then you can probably migrate! We recommend that you [reach out](mailto:support@stoplight.io) and we will schedule some time to meet and make your migration as smooth as possible.
-
-### Delete your Next Account
-
-Once you're satisfied all your projects are migrated from Next to Platform, there's an option to delete your old Next account under [account settings](https://next.stoplight.io/profile).
+While the new Stoplight Platform includes a lot of new functionality, there are some features from NEXT that have not yet made their way over. We are taking this opportunity to re-imagine and improve the functionality listed below, and will be rolling it out through the rest of the year. If any of the features below are critical to your workflow, we suggest that you wait to migrate until the relevant features are released. You can continue to use Stoplight NEXT without disruption until then.
```
Comment (marbemac) [inline 6.-migrations/next.md:64]: "If any of the features below are critical to your workflow, we suggest that you wait to migrate until the relevant features are released. You can continue to use Stoplight NEXT without disruption until then." <- suggest just removing this line at this point

File: 6.-migrations/next.md [hunk 4]
```diff
@@ -17,89 +17,68 @@ This shift leads to a number of benefits:
...
+| [~~Design: Shared parameters/responses~~](https://roadmap.stoplight.io/c/138-support-for-openapi-shared-responses) |  Launched   | Form based editing for OpenAPI shared components                                                                                          |
+| [~~Docs: Integrations~~](https://roadmap.stoplight.io/c/64-analytics-integrations)                                 |  Launched   | First class support for integrations like Google Analytics, Segment, Intercom, etc                                                        |
+| [~~Docs: Custom CSS, JS, and HTML~~](https://roadmap.stoplight.io/c/57-embeddable-component-library)               |  Launched   | Elements: a rich component library for creating custom API docs site                                                                      |
+| [Docs: Try It OAuth token generation](https://roadmap.stoplight.io/c/58-request-maker-authentication)              | Short-term  | Generate OAuth 1 & 2 tokens for HTTP requests in Try It                                                                                   |
```
Comment (marbemac) [inline 6.-migrations/next.md:76]: Should maybe update this to `considering`?

**Review by marbemac — CHANGES_REQUESTED** [review]

File: 6.-migrations/next.md (hunk 1 above)
Comment (lottamus) [reply 6.-migrations/next.md:55]:
```suggestion
- **Teams** - Stoplight is currently developing [Groups](https://roadmap.stoplight.io/c/137-workspace-groups) functionality.
```

File: 6.-migrations/next.md (hunk 2 above)
Comment (lottamus) [reply 6.-migrations/next.md:50]:
```suggestion
The NEXT Migration tool can migrate all of your NEXT projects, including all versions, OpenAPI files, Markdown files, and a single Hub file per project.
```

File: 6.-migrations/next.md (hunk 3 above)
Comment (lottamus) [reply 6.-migrations/next.md:64]:
```suggestion
While the new Stoplight Platform includes a lot of new functionality, there are some features from NEXT that have not yet made their way over. We are taking this opportunity to re-imagine and improve the functionality listed below, and will be rolling it out through the rest of the year.
```

File: 6.-migrations/next.md (hunk 4 above)
Comment (lottamus) [reply 6.-migrations/next.md:76]:
```suggestion
| [Docs: Try It OAuth token generation](https://roadmap.stoplight.io/c/58-request-maker-authentication)              | considering  | Generate OAuth 1 & 2 tokens for HTTP requests in Try It                                                                                   |
```

File: 6.-migrations/next.md (hunk 2 above)
Comment (lottamus) [reply 6.-migrations/next.md]: No, but I explain it better down below on L56: https://github.com/stoplightio/platform-docs/pull/126/files/17f14a404db4f6b09a4f26ab8a322ccc5db6c3b2#diff-b151296f24abca87088b2331db4051903503e4939d49b0b7d1f6b1d3eb7162b0R56

File: 6.-migrations/next.md (hunk 1 above)
Comment (marbemac) [reply 6.-migrations/next.md]: 🤦

**Review by marbemac — APPROVED** [review]

**Pull request merged**